README.md

```diff
@@ -1 +1,160 @@
-# Rotations
+# Rotations.jl
+
+This package implements various rotation parameterizations and defines conversions between them.  All rotation variables are stored as as immutable types.
+
+This package assumes [active (right handed) rotations](https://en.wikipedia.org/wiki/Active_and_passive_transformation) wheere applicable.
```

Comment:
I'm going for  "right handed" to indicate counter clockwise rotations.  Do you think clockwise has meaning in R3?  Cause that would be simpler.